src/Components/SampleEditor/Modules.ts

```diff
@@ -0,0 +1,207 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Please delete this file.  It shouldn't be checked in.  It should be generated at build time and then it will be ignored by .gitignore.